docker-compose.yml

```diff
@@ -36,6 +36,8 @@ services:
       - DOCKER_TAG
       - GIT_BRANCH
       - IN_DOCKER=1
+      # Uncomment to use mock USRP
+      # - MOCK=1
```

Comment:
I think we can drop this comment if we use a more descriptive envvar name, like `MOCK_RADIO`. I'd also suggest leaving it enabled but unset so that it inherits from the actual environment instead of commenting it out. That way the scos-c2 test framework can just set `MOCK_RADIO=1` in its environment and the sensor will inherit that.

Reply:
Thanks for the feedback! I implemented these changes. Please see latest version.